src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstSide;
+
+    public Square(int firstSide, Color color) {
+        this.firstSide = firstSide;

```

Comment:
Superclass' variables should be accessed through setters and getters. Avoid directly accessing superclass' variables.

Reply:
Done it.